pallets/spaces/src/lib.rs

```diff
@@ -192,7 +192,7 @@ pub mod pallet {
         ) -> DispatchResultWithPostInfo {
             let owner = ensure_signed(origin)?;
 
-            ensure!(handle_opt.is_some(), Error::<T>::HandlesAreDisabled);
+            ensure!(handle_opt.is_none(), Error::<T>::HandlesAreDisabled);
```

Comment:
Looks like we already have fixes like this in #85 already merged to [feature/move-pallets](https://github.com/dappforce/subsocial-parachain/tree/feature/move-pallets). It may conflict.

Reply:
Okay so I will merge with feature/move-pallets